Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderTitle = () => {
+	return (
+		<div class = "title">
+			<h1>Lambda School</h1>
+			<p>@LambdaSchool</p>
+			<p>26 Jan</p>
```

Comment:
Look up how to get a date in Javascript to get a date that will update accordingly.